vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -93,6 +99,33 @@ public DialogCloseActionEvent(Dialog source, boolean fromClient) {
         }
     }
 
+    /**
+     * `resize` event is sent when the user finishes resizing the overlay.
+     */
+    @DomEvent("resize") 
+    public static class DialogResizeEvent
+            extends ComponentEvent<Dialog> {
+
+        private final String width;
+        private final String height;
+
+        public DialogResizeEvent(Dialog source, boolean fromClient,
+                @EventData("event.detail.width") String width,
+                @EventData("event.detail.height") String height) {
+            super(source, fromClient);
+            this.width = width;
+            this.height = height;
+        }
+
+        public String getWidth() {
```

Comment:
Missing docs?

Reply:
Added docs for `getWidth` and `getHeight`.